libs/ledger-live-common/src/families/celo/logic.ts

```diff
@@ -0,0 +1,62 @@
+import { Account } from "../../types";
+import { CeloPendingWithdrawal, CeloValidatorGroup, CeloVote, CeloVoteStatus } from "./types";
+import { BigNumber } from "bignumber.js";
+
+export const PRELOAD_MAX_AGE = 10 * 60 * 1000;
+export const LEDGER_BY_FIGMENT_VALIDATOR_GROUP_ADDRESS =
+  "0x0861a61Bf679A30680510EcC238ee43B82C5e843";
```

Comment:
It looks like `LEDGER_BY_FIGMENT_VALIDATOR_GROUP_ADDRESS` is only used in this file, so we can omit the `export`.